### DISCUSSION TITLE
64drive - 64DD support possible?

### DISCUSSION THREAD

**Discussion by Drakonas** [Polprzewodnikowy/N64FlashcartMenu#147] [Q&A] [answered]
I am excited to see if this project could revive those with a 64Drive that have a mostly deprecated software. The device works, but development halted and now competing products actually beat the 64Drive at its own game.

I'm wondering if 64DD support would be possible with the 64Drive, since it also has native 64DD support. I am just unsure of the technicalities.

**Comment by Polprzewodnikowy** [comment, marked as answer]
64drive does not have native 64DD support in the firmware. It's not possible to use unmodified game disk dumps with the 64drive. 64drive's author would need to release new firmware with support for that feature. Loading unmodified 64DD disk images is reserved for the SummerCart64 only at the moment.

**Reply by Drakonas** [reply]
Do you mean actual .ndd disk image support? Because cartridge ports work fine on the 64Drive. Not sure if I'm understanding if the cartridge ports are intended to run entirely independent of 64DD.

To be honest I thought that Marshall made the 64Drive to be intended for 64DD development since the debug features can integrate with 64DD, so I was kind of shocked at your response. But I realize now I may have misunderstood the tech.

**Reply by Polprzewodnikowy** [reply]
SC64 has 64DD emulated in the hardware and it acts as if you have 64DD plugged to the bottom of the console. 64drive has none of that. Cartridge ports, as the name suggests, are modifications to the original disk dumps to remove the need of 64DD hardware. Cartridge ports and `.ndd` disk dumps are completely different thing - ports were made because of the lack of 64DD hardware emulation in the flashcarts. SummerCart64's different approach to the hardware design removes the need of use of the cartridge ports because it can support 64DD disk dumps directly.

**Reply by Drakonas** [reply]
I realized I answered my own question. Apologies for taking your time. Thanks for spending the time to answer. I'll look into a SC64 at some point then. Thank you!
